kafka-3.8.yaml

```diff
@@ -73,6 +73,8 @@ subpackages:
           ln -s /usr/lib/kafka/libs ${{targets.subpkgdir}}/opt/bitnami/kafka/libs
           ln -s /usr/lib/kafka/config ${{targets.subpkgdir}}/opt/bitnami/kafka/config
           ln -s /usr/lib/kafka/logs ${{targets.subpkgdir}}/opt/bitnami/kafka/logs
+          mv ${{targets.destdir}}/usr/lib/kafka/config/server.properties ${{targets.destdir}}/usr/lib/kafka/config/server.properties.original
+          mkdir ${{targets.destdir}}/docker-entrypoint-initdb.d/
```

Comment:
the directory seems specific to `bitnami-compat` I think we should use `${{targets.contextdir}}` here.